week1/homework/src/calculations.js

```diff
@@ -0,0 +1,26 @@
+function calculator(url, code, response) {
+  response.setHeader('Content-Type', 'text/html');
+  response.statusCode = code;
+  let state = 10;
+  if (url === '/') {
```

Comment:
You can use `if`, so this is a good solution, but just as a suggestion, you could also use `case switch`, which can make it more readable sometimes, in this example you would have: 
```
switch (url) {
    case '/':
        response.write('You can use "/state" to see the current state.');
        response.write('You can use "/add" to add 1 to the current state.');
        response.write('You can use "/subtract" to subtract 1 from the current state.');
        response.write('You can use "/reset" to reset the current state.');
        break;
    case '/state':
        response.write('The current state is ' + state);
        break;
    case '/add':
        response.write('The current state is ' + ++state);
        break;
    case '/subtract':
        response.write('The current state is ' + --state);
        break;
    case '/reset':
        state = 10;
        response.write('The current state is ' + state);
        break;
    default:
        response.write('Not found.');
}
```